vendor/github.com/containers/image/docker/docker_image_src.go

```diff
@@ -51,13 +52,9 @@ func newImageSource(ctx *types.SystemContext, ref dockerReference, requestedMani
 	if !acceptableRequestedMIMEs {
 		requestedManifestMIMETypes = manifest.DefaultRequestedManifestMIMETypes
 	}
-	return &dockerImageSource{
-		ref: ref,
-		requestedManifestMIMETypes: requestedManifestMIMETypes,
-		c: c,
-	}, nil
+	return                                               &dockerImageSource{ref: ref, requestedManifestMIMETypes: requestedManifestMIMETypes, c: c}, nil
```

Comment:
was this a gofmt change?  Seems off...

Reply:
That's a funny one. Thanks for checking! Do you want me to clean that up?

Reply:
If you've the energy to do so, then yes, but not a biggy.

Btb are you making the same changes in the docker-1.13.1 branch too?

Reply:
Do we support the non-rhel `docker-1.13.1` as well?
/me cries in silence

Reply:
Ah. Note that this is against `docker-1.13.1-rhel` (not master).

Reply:
we've made changes in both docker-1.13.1 and docker-1.13.1-rhel in lock step AFAIK.  But perhaps we can drop the docker-1.13.1 going forward, but that's a @rhatdan decision.

Reply:
docker-1.13.1 is for Fedora 30